draft-ietf-quic-transport.md

```diff
@@ -937,20 +937,17 @@ connection ID for the duration of the connection or until its peer invalidates
 the connection ID via a RETIRE_CONNECTION_ID frame
 ({{frame-retire-connection-id}}).
 
-Endpoints store received connection IDs for future use.  An endpoint that
-receives excessive connection IDs MAY discard those it cannot store.  An
-endpoint that issues connection IDs cannot expect its peer to store and use all
-issued connection IDs.
-
 An endpoint SHOULD ensure that its peer has a sufficient number of available and
-unused connection IDs.  While each endpoint independently chooses how many
-connection IDs to issue, endpoints SHOULD provide and maintain at least eight
-connection IDs.  The endpoint can do this by always supplying a new connection
-ID when a connection ID is retired by its peer or when the endpoint receives a
-packet with a previously unused connection ID.  Endpoints that initiate
-migration and require non-zero-length connection IDs SHOULD provide their peers
-with new connection IDs before migration, or risk the peer closing the
-connection.
+unused connection IDs. Endpoints store received connection IDs for future use. 
+They advertise the limit how many unretired connection IDs they are willing to 
```

Comment:
Word missing?

Reply:
Where?

Reply:
"advertise the limit how many" could probably do with a limit "on" how many, but it might read better to simply "advertise how many".

Reply:
Thanks for the suggestion, @MikeBishop!